tests/integration/api/test_xai.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from pathlib import Path
+
+import numpy as np
+import openvino.runtime as ov
+import pytest
+from otx.core.data.entity.base import OTXBatchPredEntityWithXAI
+from otx.core.data.entity.classification import (
+    MulticlassClsBatchPredEntity,
+    MulticlassClsBatchPredEntityWithXAI,
+)
+from otx.engine import Engine
+
+RECIPE_LIST_ALL = pytest.RECIPE_LIST
+MULTI_CLASS_CLS = [recipe for recipe in RECIPE_LIST_ALL if "multi_class_cls" in recipe]
+MULTI_LABEL_CLS = [recipe for recipe in RECIPE_LIST_ALL if "multi_label_cls" in recipe]
+MC_ML_CLS = MULTI_CLASS_CLS + MULTI_LABEL_CLS
+
+DETECTION_LIST = [recipe for recipe in RECIPE_LIST_ALL if "/detection" in recipe and "tile" not in recipe]
+INST_SEG_LIST = [recipe for recipe in RECIPE_LIST_ALL if "instance_segmentation" in recipe and "tile" not in recipe]
+EXPLAIN_MODELS = MC_ML_CLS + DETECTION_LIST + INST_SEG_LIST
+
+MEAN_TORCH_OV_DIFF = 150
+
+
+@pytest.mark.parametrize(
+    "recipe",
+    MULTI_CLASS_CLS,
```

Comment:
`MULTI_CLASS_CLS` -> `EXPLAIN_MODELS`?

Reply:
Will have to update this asseqrt condition with proper types but true:
`assert isinstance(predict_result[0], MulticlassClsBatchPredEntity)`
`assert isinstance(predict_result_explain[0], MulticlassClsBatchPredEntityWithXAI)`

Reply:
Indeed, this needs to be updated